src/shuffle.ts

```diff
@@ -0,0 +1,41 @@
+import { purry } from './purry';
+
+/**
+ * Creates an array of shuffled values, using the
+ * [Fisher-Yates shuffle](https://en.wikipedia.org/wiki/Fisher-Yates_shuffle).
```

Comment:
No need to document the algorithm used, that is an internal that isn't relevant to the caller, and would become stale if we ever implement a different shuffling algorithm.